src/msg/async/AsyncConnection.cc

```diff
@@ -863,6 +863,7 @@ ssize_t AsyncConnection::_process_connection()
         }
 
         SocketOptions opts;
+        opts.priority = async_msgr->get_socket_priority();
```

Comment:
previously we set priority on AsyncMessenger::accept, but now it's move to PosixStack.cc. So we need to pass priority value here